assets/tests/components/nav/topNav.test.tsx

```diff
@@ -68,47 +68,53 @@ describe("TopNav", () => {
       )
 
       expect(
-        result.queryByRole("button", { name: "User Info" })
+        screen.queryByRole("button", { name: "User Info" })
       ).toBeInTheDocument()
     })
 
-    test("brings up an element with 'logged in as' text when clicked", async () => {
+    test("brings up an element with 'logged in as' text and a logout ling when clicked", async () => {
       jest.mocked(getTestGroups).mockReturnValue([TestGroups.KeycloakSso])
 
       const dispatch = jest.fn()
       const user = userEvent.setup()
-      const result = render(
+      render(
         <StateDispatchProvider state={initialState} dispatch={dispatch}>
           <BrowserRouter>
             <TopNav />
           </BrowserRouter>
         </StateDispatchProvider>
       )
 
-      expect(result.queryByText("Logged in as")).not.toBeInTheDocument()
+      expect(screen.queryByText("Logged in as")).not.toBeInTheDocument()
 
-      await user.click(result.getByTitle("User Info"))
+      await user.click(screen.getByTitle("User Info"))
```

Comment:
**praise**: Appreciate replacing `result` with `screen` here! I think I goofed that when I added these tests, and this is helpful to give me a better sense of when `screen` is usable (I'm now thinking more like... always 🤯)